functions/go/apply-setters/applysetters/apply_setters_test.go

```diff
@@ -390,11 +390,43 @@ data:
 			expectedResources: `apiVersion: v1
 kind: Service
 metadata:
-  name: # kpt-set: ${app}
-  namespace: # kpt-set: ${ns}
+  name: null # kpt-set: ${app}
+  namespace: null # kpt-set: ${ns}
+image: :1.7.1 # kpt-set: ${image}:${tag}
+env: # kpt-set: ${env}
+  - null
+`,
+		},
+		{
+			name: "set non-empty values from empty values",
+			input: `apiVersion: v1
+kind: Service
+metadata:
+  name: null # kpt-set: ${app}
```

Comment:
As a user of this function this would be unexpected for me, I would have expected the empty string here. Providing the empty string as a setter value and having `null` set seems strange to me.

Reply:
Updated to empty string with DoubleQuotes.